semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awareruby-sca.yaml-dependency_awaresca.rb/results.json

```diff
@@ -30,7 +30,7 @@
               "resolved_url": null,
               "version": "1.21.0"
             },
-            "lockfile": "Gemfile.lock"
+            "lockfile": "targets/dependency_aware/Gemfile.lock"
```

Comment:
Did you intend to change the path of the lockfile in the json output here @mmcqd

Reply:
I did. Should have included it in the PR description, my bad. Updating it